### PR TITLE
Documentation for additional regions AWS / Azure

### PR DESCRIPTION
**Pull request by agrare**
Closes https://github.com/ManageIQ/manageiq-documentation/issues/1614

### CODE REVIEW

File: managing_providers/_topics/adding_amazon_cloud_regions.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+the appliance server. You can use this capability to add new regions that have been
+set up since {{ site.data.product.title_short }} was released.  Once adding the region it will be
+available when creating a new Amazon EC2 provider.
+
+1.  Click ![config gear](../images/config-gear.png) (**Configuration**).
+
+2.  Click on the **Settings** accordion, then click **Zones**.
+
+3.  Click the zone where the {{ site.data.product.title_short }} server is located,
+    then click on the EVM server.
```
Comment (Fryguy) [inline managing_providers/_topics/adding_amazon_cloud_regions.md:13]: Maybe we should make it a region setting?

Comment (agrare) [reply managing_providers/_topics/adding_amazon_cloud_regions.md:13]: I mirrored the settings from "disabling a region"  If we change it here we should change it there also, you want to change in both spots?

File: managing_providers/_topics/adding_azure_cloud_regions.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+3.  Click the zone where the {{ site.data.product.title_short }} server is located,
+    then click on the EVM server.
+
+4.  Click on the **Advanced** tab.
+
+5.  Search for `:ems_azure:`, and enter the regions you want to add
+    under `:additional_regions:`.
+
+        Example. To add hypothetical `eastus4` and `eastus4` regions:
```
Comment (Fryguy) [inline managing_providers/_topics/adding_azure_cloud_regions.md:20]:
```suggestion
        Example. To add hypothetical `eastus4` and `eastus5` regions:
```

Even so, I like the lunar base example better - can we make these lunar base?

Comment (agrare) [reply managing_providers/_topics/adding_azure_cloud_regions.md:20]: As soon as Nadella also starts a personal space program :wink:

File: managing_providers/_topics/adding_amazon_cloud_regions.md (hunk 1 above)
Comment (agrare) [reply managing_providers/_topics/adding_amazon_cloud_regions.md]: Done

File: managing_providers/_topics/adding_azure_cloud_regions.md (hunk 2 above)
Comment (agrare) [reply managing_providers/_topics/adding_azure_cloud_regions.md]: Done

File: managing_providers/_topics/adding_amazon_cloud_regions.md (hunk 1 above)
Comment (Fryguy) [reply managing_providers/_topics/adding_amazon_cloud_regions.md]: yeah i think in both places makes sense.

**Review by Fryguy — APPROVED** [review]

**Comment by Fryguy** [comment]
Backported to `morphy` in commit 2abd0e49876a3347243504b826deb551113a115a.

```text
commit 2abd0e49876a3347243504b826deb551113a115a
Author: Jason Frey <fryguy9@gmail.com>
Date:   Mon Nov 22 15:55:20 2021 -0500

    Merge pull request #1615 from agrare/add_docs_adding_additional_regions
    
    Documentation for additional regions AWS / Azure
    
    (cherry picked from commit 0c10ac0dd3b164d1f7b683533717b3c9989c1f13)
```

**Pull request merged**